homeassistant/components/alphaess/strings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "config": {
+    "flow_title": "Alpha ESS: {username}",
```

Comment:
I don't think this extra username hint is required. I would just remove the flow title to keep it simple.